src/main/resources/templates/lsp/stylelint-lsp/settings.schema.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Added JSON schema for config based on [this](https://github.com/bmatcuk/stylelint-lsp?tab=readme-ov-file#settings).